bitnami/common/Chart.yaml

```diff
@@ -1,23 +1,22 @@
+annotations:
+  category: Infrastructure
 apiVersion: v1
-name: common
-# Please make sure that version and appVersion are always the same.
-version: 0.7.1
-appVersion: 0.7.1
-description: A Library Helm Chart for grouping common logic between bitnami charts. This chart is not deployable by itself.
+appVersion: 0.8.0
+description: A Library Helm Chart for grouping common logic between bitnami charts.
+  This chart is not deployable by itself.
+home: https://github.com/bitnami/charts/tree/master/bitnami/common
+icon: https://bitnami.com/downloads/logos/bitnami-mark.png
 keywords:
   - common
   - helper
   - template
   - function
   - bitnami
-home: https://github.com/bitnami/charts/tree/master/bitnami/common
-icon: https://bitnami.com/downloads/logos/bitnami-mark.png
+maintainers:
+  - email: containers@bitnami.com
+    name: Bitnami
+name: common
 sources:
   - https://github.com/bitnami/charts
   - http://www.bitnami.com/
-maintainers:
-  - name: Bitnami
-    email: containers@bitnami.com
-engine: gotpl
-annotations:
-  category: Infrastructure
+version: 0.8.0
```

Comment:
Please could you add this line `# Please make sure that version and appVersion are always the same.`